src/scopes/activityReport/role.js

```diff
@@ -0,0 +1,21 @@
+import { Op } from 'sequelize';
+import { sequelize } from '../../models';
+
+export function withRole(roles) {
+  const rolesString = roles.map((role) => `'${role}'`).join(',');
+  return {
+    [Op.or]: [
+      sequelize.literal(`SELECT ARRAY((SELECT unnest("role") FROM "Users" where "ActivityReport"."userId" = "Users"."id"))::text[] && ARRAY[${rolesString}]`),
```

Comment:
Is there a way to rework this so that `rolesString` is not being inserted directly into the SQL here (using parameters or similar?). If `roles` is coming from user input there's a risk of SQL injection here.